.github/workflows/build-images.yml

```
@@ -223,8 +223,13 @@ jobs:
         run: |
           rm -rf "scripts/ci"
           mv "main-airflow/scripts/ci" "scripts"
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+          cache: 'pip'
+      - run: python -m pip install --editable ../../dev/breeze/
```

```suggestion
          cache: 'pip'
          cache-dependency-path: ./dev/breeze
      - run: python -m pip install --editable ./dev/breeze/
```

I believe it should be set like that everywhere @edithturn

---

See examples here: https://github.com/actions/setup-python#caching-packages-dependencies

---

(Oh how I wish YAML anchors worked here.)

---

> (Oh how I wish YAML anchors worked here.)

oh, absolutely

---

> (Oh how I wish YAML anchors worked here.)

oh, absolutely

---

2 years and 87 comments: https://github.community/t/support-for-yaml-anchors/16128/88